web_session_allow_public/demo/web_session_allow_public_demo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2016 LasLabs Inc.
+    @license AGPL-3 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+
+<odoo>
+    <template id="assets_frontend" name="Public Session Demo" inherit_id="website.assets_frontend">
+        <xpath expr="." position="inside">
+            <script src="/web_session_allow_public/static/src/js/web_session_allow_public.demo.js" />
```

Comment:
Yup you're right, we definitely want that. I'm going to import in the real assets instead of demo though.